### DISCUSSION TITLE
CloudWatch log derived field to Tempo trace

### DISCUSSION THREAD

**Discussion by berabulut** [grafana/grafana#42498] [Data Sources]
I am manually instrumenting my Lambda functions and sending traces to Tempo. With the query below I can access CloudWatch logs and can see traceID as a derivedField in Grafana.

`fields @timestamp, @message | filter @message like /traceID/ | sort @timestamp desc`

I want to go a Tempo trace directly with a button like Loki's. Is this possible?

**Comment by AlexRex** [comment]
Hi, any news on this? @berabulut Did you achieve this?

**Reply by berabulut** [reply]
Hi @AlexRex , I couldn't find a way to achieve this.

**Comment by sarahzinger** [comment]
Hey folks, thanks for opening this discussion! As part of our work [to move from discussions back to issues](https://github.com/grafana/grafana/discussions/73424), I created an [issue](https://github.com/grafana/grafana/issues/77948) for this feature request and put it in our backlog to consider for future work.

I think the suggestion to have more ways to connect the cloudwatch datasource to other datasources in grafana is a great one! Will close this discussion for now, but if you have other thoughts on what you're like to see with this feature, feel free to add it to the issue above!
